weave/trace_server/sqlite_trace_server.py

```diff
@@ -206,6 +206,10 @@ def call_start(self, req: tsi.CallStartReq) -> tsi.CallStartRes:
             trace_id=req.start.trace_id,
         )
 
+    # TODO: The Sqlite Trace Server does not support batching.
+    def call_start_batch(self, req: tsi.CallCreateBatchReq) -> tsi.CallCreateBatchRes:
+        return tsi.CallCreateBatchRes(res=[])
```

Comment:
it is easy: just loop through

Reply:
If ok with you, I prefer to implement in a follow up PR for all of the relevant servers at once.